src/resolvers/line_items.ts

```diff
@@ -13,12 +13,23 @@ import {
   LineItemStatus,
   LineItemUpdateInput,
   MutationResolvers,
+  QueryResolvers,
 } from '../server-internal-types'
 import getPalletWithParentAssociations from './getPalletWithParentAssociations'
 import { authorizeOfferMutation } from './offer_authorization'
 import validateEnumMembership from './validateEnumMembership'
 import validateUris from './validateUris'
 
+const lineItem: QueryResolvers['lineItem'] = async (_, { id }, context) => {
+  const lineItem = await getLineItemWithParentAssociations(id)
+
+  if (!lineItem) {
+    throw new UserInputError(`LineItem ${id} does not exist`)
+  }
+
```

Comment:
issue (blocking): this query needs to be authorized. Either authorize it using [`authorizeOfferQuery`](https://github.com/distributeaid/shipment-tracker/blob/saga/src/resolvers/offer_authorization.ts#L83) (you're already using `getLineItemWithParentAssociations` so the offer is available) or file an issue for authorizing it and I can pick it up.